Webpack.NET/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.AspNet.Mvc" version="5.2.3" targetFramework="net452" />
-  <package id="Microsoft.AspNet.Razor" version="3.2.3" targetFramework="net452" />
-  <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net452" />
-  <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net452" />
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
+  <package id="Microsoft.AspNet.Mvc" version="5.2.3" targetFramework="net45" />
+  <package id="Microsoft.AspNet.Razor" version="3.2.3" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net45" />
+  <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="10.0.2" targetFramework="net45" />
```

Comment:
After downgrading the target framework to .NET 4.5 I had to do a `Update-Package -Reinstall`, so that probably updated all NuGet references. Projects can use the higher version and use an assembly binding redirect, so no problem here (https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/redirect-assembly-versions). I'm not totally sure, but as far as I can remember, installing the `Newtonsoft.Json` package already adds this for you.